src/core/connection-parameters.cpp

```diff
@@ -119,6 +119,16 @@ connection_parameters::connection_parameters(connection_parameters const& other)
         backendRef_->inc_ref();
 }
 
+connection_parameters::connection_parameters(connection_parameters && other)
+    : factory_(std::move(other.factory_)),
```

Comment:
Having the std::move in place even for raw primitive types makes it easier to refactor in the future. It's easy to overlook the move code and it would introduce a silent copy. But as you like. Please let me know if you don't want std::move for primitive types. I'll change them all at once.

Reply:
OK, I don't have a strong enough opinion about this to ask you to change it if you consciously prefer the current way. I recognize the merit of your argument but OTOH I also think that seeing a useless `move()` is a real drawback. But, again, let's leave it like this if you like it better.